release_notes/ocp-4-3-release-notes.adoc

```diff
@@ -2942,3 +2942,59 @@ After this one-time manual restart, subsequent upgrades and rotations will
 ensure restart before the service CA expires without requiring manual
 intervention.
 ====
+
+[[ocp-4-3-31]]
+=== RHBA-2020:3180 - {product-title} 4.3.31 Bug Fix Update
+
+Issued: 2020-08-05
+
+{product-title} release 4.3.31 is now available. The container images and bug
+fixes in the update are documented in the
+link:https://access.redhat.com/errata/RHBA-2020:3180[RHBA-2020:3180] advisory.
+The list of packages included included in the update are provided by the
+link:https://access.redhat.com/errata/RHBA-2020:3179[RHBA-2020:3179] advisory.
+
+Space precluded documenting all of the container images for this release in the
+advisory. See the following article for notes on the container images in this
+release:
+
+link:https://access.redhat.com/solutions/5286571[{product-title} 4.3.31 container image list]
+
+[[ocp-4-3-31-upgrading]]
+==== Upgrading
+
+To upgrade an existing {product-title} 4.3 cluster to this latest release, see
+xref:../updating/updating-cluster-cli.adoc#update-service-overview_updating-cluster-cli[Updating a cluster by using the CLI for instructions.]
+
+[IMPORTANT]
+====
+If you are upgrading to this release from {product-title} 4.2 or {product-title}
+4.3.3 or earlier, you must restart all Pods after the upgrade is complete.
+
+This is because the service CA is automatically rotated as of {product-title}
+4.3.5. The service CA is rotated during the upgrade and a restart is required
+afterward to ensure that all services are using the new service CA before the
+previous service CA expires.
+
+After this one-time manual restart, subsequent upgrades and rotations will
+ensure restart before the service CA expires without requiring manual
+intervention.
+====
+
+[[RHSA-2020:3183]]
+=== RHSA-2020:3183 - Moderate: {product-title} 4.3 Security Update
+
+Issued: 2020-08-05
+
+An update for openshift is now available for {product-title} 4.3. Details of
```

Comment:
There's another use of "openshift" in this way in the release notes, so it's probably fine. I am curious how this usage came to be, though.

Reply:
It's the component listed in the RHSA - this is and "kubernetes" are the biggest culprits for why it makes sense to start putting these in backticks so it's clear they're referring to something other than the product name. If we agree on that in the template review I'm happy to do a search and fix in the Release Notes for currently supported versions :)